api/CSSGroupingRule.json

```diff
@@ -47,6 +47,153 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "cssRules": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSGroupingRule/cssRules",
+          "spec_url": "https://drafts.csswg.org/cssom/#dom-cssgroupingrule-cssrules",
+          "support": {
+            "chrome": {
+              "version_added": "45"
```

Comment:
This could be correct, but due to https://github.com/mdn/browser-compat-data/issues/7844 I'm suspicious of any Chrome 45 versions. What testing was done to get this version? There are generated tests for this in mdn-bcd-collector, but they were broken. I've sent https://github.com/foolip/mdn-bcd-collector/pull/1323 to fix it and then it should be possible to use these tests:
https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/CSSGroupingRule

Reply:
I copied the data from the main interface (that I haven't changed), after searching to see if there was any indication that this had been partly implemented. So if it's wrong, the main interface is likely wrong too.

Reply:
Looks like this came from https://bugs.chromium.org/p/chromium/issues/detail?id=496381, and the date *does* line up with Chrome 45. I've confirmed that the `CSSGroupingRule` is there in Chrome 45, but not in 44.

That just shuffled the prototype chain around a little bit and is a case of https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#apis-moved-on-the-prototype-chain, but sorting that out need not block this PR. I'll review.